src/scss/fotorama.scss

```diff
@@ -18,7 +18,7 @@ $blue: mix(#00d1ff, #008ed6);
   }
 }
 
-.fullscreen {
+.fotorama__fullscreen {
```

Comment:
because now we will have two similar classes: `.fotorama--fullscreen` and `.fotorama__fullscreen`